src/docs/development/add-to-app/ios/add-flutter-screen.md

```diff
@@ -0,0 +1,487 @@
+This is what the Flutter module template provides. Executing
+`flutter create -t module xxx` produces a Flutter project containing a CocoaPods
+pod designed for consumption by your existing host app.
+
+Let's assume you have an existing iOS app at `some/path/MyApp`, and that you
+want your Flutter project as a sibling:
+
+```bash
+cd some/path/
+flutter create -t module my_flutter
+```
+
+This creates a `some/path/my_flutter/` Flutter module project with some Dart
+code to get you started and a `.ios/` hidden subfolder that wraps up the
+module project that contains some Cocoapods and a helper Ruby script.
+
+## Make the host app depend on the Flutter module
+
+The description below assumes that your existing iOS app has a structure similar
+to what you get by asking Xcode version 10.0 to generate a new "Single View App"
+project using Objective-C. If your existing app has a different folder structure
+and/or existing `.xcconfig` files, you can reuse those, but probably need to adjust
+some of the relative paths mentioned below accordingly.
+
+The assumed folder structure is as follows:
+
+```text
+some/path/
+  my_flutter/
+    lib/main.dart
+    .ios/
+  MyApp/
+    MyApp/
+      AppDelegate.h
+      AppDelegate.m (or swift)
+      :
+```
+
+### Add your Flutter app to your Podfile
+
+Integrating the Flutter framework requires use of the CocoaPods dependency manager.
+This is because the Flutter framework needs to be available also to any Flutter plugins
+that you might include in my_flutter.
+
+Please refer to [cocoapods.org](https://cocoapods.org/) for how to install CocoaPods
+on your development machine, if needed.
+
+If your host application (`MyApp`) is already using CocoaPods, you only have to do the
+following to integrate with your `my_flutter` app:
+
+1. Add the following lines to your `Podfile`:
+
+```ruby
+  flutter_application_path = 'path/to/my_flutter/'
+  load File.join(flutter_application_path, '.ios', 'Flutter', 'podhelper.rb')
+```
+
+2. For each Xcode [target](https://guides.cocoapods.org/syntax/podfile.html#target) that needs to embed Flutter, call `install_all_flutter_pods(flutter_application_path)`.
+
+```ruby
+  target 'MyApp' do
+    install_all_flutter_pods(flutter_application_path)
+  end
+  target 'MyAppTests' do
+    install_all_flutter_pods(flutter_application_path)
+  end
+```
+
+3. Run `pod install`.
+
+Whenever you change the Flutter plugin dependencies in `some/path/my_flutter/pubspec.yaml`,
+you need to run `flutter pub get` from `some/path/my_flutter` to refresh the list
+of plugins read by the `podhelper.rb` script. Then run `pod install` again from
+`some/path/MyApp`.
+
+The `podhelper.rb` script will ensure that your plugins, the Flutter.framework, and the App.framework
+are embedded in your project.
+
+You should now be able to build the project using `⌘B`.
+
+### Under the hood
+
+If you have some reason to do this manually or debug why these steps aren't working, here's what's going on under the hood:
```

Comment:
Please use shorter lines. Also add a sentence:
"If you have some reason... Otherwise, skip this section."

Reply:
I'm not sure how to do that while preserving the colon =(